CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+
+Couchbase welcomes and encourages community contributions to our Capella provider! Below are the following steps for making a contribution to this provider:
+
+## Prerequisites
+
+### Enable MFA
+
+We require all contributors have [MFA enabled](https://help.github.com/en/github/authenticating-to-github/configuring-two-factor-authentication) on their account.
+
+### Verify your commits
+
+We require that contributors verify their commits. You’ll need to do this before GitHub will accept pushes for all branches.
+Please follow the steps below:
+
+1. [Generate a GPG Key](https://help.github.com/en/github/authenticating-to-github/generating-a-new-gpg-key)
+2. [Add your public key to your Github Account](https://help.github.com/en/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account)
+3. [Tell your local git about your key](https://help.github.com/en/github/authenticating-to-github/telling-git-about-your-signing-key)
+
+```
+$ git config user.signingkey YOUR_KEY
+```
+
+4. [Configure git to sign your commits](https://help.github.com/en/github/authenticating-to-github/signing-commits) for the local repository
+
+```
+$ git config commit.gpgsign true
+```
+
+5. Make sure your [local git is configured to use the same email address](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address) as your GPG key.
+
+```
+$ git config user.email "your.name@email.com"
+```
+
+## Before Contributing
+
+<!-- Sign the contributing agreement -->
+
+1. Please read through the [Terraform Contribution Guidelines](https://www.terraform.io/docs/extend/community/contributing.html) and the [README](https://github.com/couchbasecloud/terraform-provider-couchbasecapella/blob/main/README.md) in this repository.
+2. Please [file an issue](https://github.com/couchbasecloud/terraform-provider-couchbasecapella/issues) for the contribution you'd like to make, where we can then discuss.
+
+## After Issue Discussion
+
+Once we've discussed your proposed contribution, and you're ready to implement your changes, please follow the following steps:
+
```

Comment:
I think we should call out the need to Fork the repo first here.

Reply:
I'll add that in now